packages/api/start/hooks.js

```diff
@@ -44,5 +44,75 @@ hooks.after.providersBooted(() => {
 		}
 	};
 
+	const repeated = [
+		'11111111111',
+		'22222222222',
+		'33333333333',
+		'44444444444',
+		'55555555555',
+		'66666666666',
+		'77777777777',
+		'88888888888',
+		'99999999999',
+	];
+
+	const cpfFn = async (data, field, message, args, get) => {
+		const value = get(data, field);
+
+		if (!value) {
+			// required rule should take care
+			return;
+		}
+
+		if (!/^[\d-.]+$/.test(value)) {
+			// accepts only digits, - and .
+			throw message || `invalid cpf (${value}).`;
```

Comment:
Eu vi que você incluiu lá no `validator.json` uma opção de cpf, correto? No caso aquela tradução está sendo utilizada? Se for o caso, seria interessante traduzir o texto, assim como está sendo feito de um modo geral usando o `antl`. Então, aqui poderia usar algo como `antl(validation.cpf)`, pra manter a tradução, visto que logo (acho que tem uma issue aberta) iremos implementar uma forma de o usuário poder modificar a linguagem não só do front, mas também do backend.

Reply:
Luiz por algum motivo ele não mostra as mensagem que são lançadas na função do CPF, em todos os erros ele apenas usa o que está em validation.json, e nesse caso já está traduzido:
```javascript
{
  "error": {
    "error_code": "VALIDATION_ERROR",
    "message": [
      {
        "message": "cpf é inválido ",
        "field": "cpf",
        "validation": "cpf"
      }
    ]
  }
}
```
Se a função for usada de forma unitária ai tem como capturar os erros.